apps/aehyperchains/include/aehc_types.hrl

```diff
@@ -0,0 +1,37 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
+
+-include_lib("aecore/include/blocks.hrl").
+-include_lib("aecontract/include/hard_forks.hrl").
+
+-define(STAKING_CONTRACT, {?MODULE, staking_contract}).
+-define(STAKING_CONTRACT_ADDR, {?MODULE, staking_contract_addr}).
+%% Lima or Iris as we need the FATE VM at genesis
+%% In case that's unwanted then start up another consensus before hyperchains
+-define(HC_GENESIS_VERSION, ?LIMA_PROTOCOL_VSN).
+
+%% Magic nonces
+-define(NONCE_HC_ENABLED, 16#ffffffffffffffff - 1).
+-define(NONCE_HC_POGF, 16#ffffffffffffffff).
+-define(NONCE_HC_GENESIS, 2). %% Hyperchain at genesis :)
+
+-define(COMMITER_PUB_BYTES, 32).
+-define(POGF_HASH_BYTES, 32).
+-define(COMMITMENT_HASH_BYTES, 32).
+
+-type commiter_pubkey() :: <<_:(?COMMITER_PUB_BYTES*8)>>.
+-type pogf_hash() :: <<_:(?POGF_HASH_BYTES*8)>>.
+-type commitment_hash() :: <<_:(?COMMITMENT_HASH_BYTES*8)>>.
+
+-type pubkey() :: aec_keys:pubkey().
```

Comment:
I don't really like putting things here that have nothing todo with hc. Especially non trivial definition like signature() or query()